extensions/auth/portability-auth-facebook/src/main/java/org/datatransferproject/auth/facebook/FacebookOAuthConfig.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2018 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.auth.facebook;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import java.util.List;
+import java.util.Map;
+import org.datatransferproject.auth.oauth2.OAuth2Config;
+
+public class FacebookOAuthConfig implements OAuth2Config {
+
+  @Override
+  public String getServiceName() {
+    return "Facebook";
```

Comment:
Is this for display or a canonical value that should be all in the same case?

Reply:
I believe it's functionally only for display, aside from the client ids and secrets (which are the service name in all-caps)